### ISSUE TITLE
getting tons of ignoredMessages.includes is not a function errors

### ISSUE THREAD

**Issue by fiws** [AtomLinter/linter-eslint#615]
```
TypeError: ignoredMessages.includes is not a function
    at /home/fiws/dotfiles/atom.symlink/packages/linter-eslint/lib/worker.js:37:29
    at Array.filter (native)
    at lintJob (/home/fiws/dotfiles/atom.symlink/packages/linter-eslint/lib/worker.js:36:42)
    at /home/fiws/dotfiles/atom.symlink/packages/linter-eslint/lib/worker.js:69:20
    at Emitter.emit (/home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/node_modules/sb-event-kit/lib/emitter.js:70:19)
    at /home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:22:23
    at /home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:21:24
    at Emitter.emit (/home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/node_modules/sb-event-kit/lib/emitter.js:68:19)
    at Communication.parseMessage (/home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:63:18)
    at process.ProcessCommunication.messageCallback (/home/fiws/dotfiles/atom.symlink/packages/linter-eslint/node_modules/process-communication/lib/communication.js:35:27)
```

Note that I am currently stuck with atom 1.6.2 so that might be break something?

**Comment by mrx88** [comment]
+ 1

**Comment by mrx88** [comment]
NOTE: upgrading atom from 1.6.2 to 1.8.0 fixed this issue for me.

**Comment by srahulprdxn** [comment]
+1 Even I am getting this error.

```
TypeError: ignoredMessages.includes is not a function
    at /home/rahul/.atom/packages/linter-eslint/lib/worker.js:37:29
    at Array.filter (native)
    at lintJob (/home/rahul/.atom/packages/linter-eslint/lib/worker.js:36:42)
    at /home/rahul/.atom/packages/linter-eslint/lib/worker.js:69:20
    at Emitter.emit (/home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/node_modules/sb-event-kit/lib/emitter.js:70:19)
    at /home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:22:23
    at /home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:21:24
    at Emitter.emit (/home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/node_modules/sb-event-kit/lib/emitter.js:68:19)
    at Communication.parseMessage (/home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/node_modules/sb-communication/lib/main.js:63:18)
    at process.ProcessCommunication.messageCallback (/home/rahul/.atom/packages/linter-eslint/node_modules/process-communication/lib/communication.js:35:27)
```

**Comment by srahulprdxn** [comment]
But as @mrx88 has commented, upgrading atom to 1.8.0 solved the issue.

**Comment by Arcanemagus** [comment]
Duplicate of #614.

**Comment by Arcanemagus** [comment]
@fiws I'm not sure how one gets "stuck" with an out of date Atom, but if you are truly unable to update you can install an older version of the package with `apm install linter-eslint@x.y.z`.

**Comment by srahulprdxn** [comment]
The reason one might get stuck with atom is 'coz atom doesn't send an update notification or update the editor by itself if people didn't install it from package managers.

**Comment by fiws** [comment]
probably true, but not in my case ;)

https://github.com/atom/atom/issues/10918 is my problem. Atom crashes on Arch until 1.9 (they upgraded electron to fix this bug in 1.9)
So I am literally stuck with it unless I decide to go beta or get Atom from source.

**Comment by srahulprdxn** [comment]
@fiws Aah got that. Thanks for sharing.

**Comment by stowball** [comment]
I'm getting this too because I'm scared of upgrading to 1.8. 1.7 basically cost me hours of productivity losses because it ran like a dog and constantly hung. Nothing in the changelog suggested it would fix any of that, so I stuck with 1.6.

I guess I now have to make a decision: downgrade es-lint, or upgrade Atom 😞
